src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Create.cs

```diff
@@ -8,141 +8,103 @@
     public static partial class ZipFile
     {
-        /// <summary>
-        /// Opens a <code>ZipArchive</code> on the specified path for reading. The specified file is opened with <code>FileMode.Open</code>.
-        /// </summary>
-        ///
-        /// <exception cref="ArgumentException">archiveFileName is a zero-length string, contains only whitespace, or contains one
-        ///                                     or more invalid characters as defined by InvalidPathChars.</exception>
-        /// <exception cref="ArgumentNullException">archiveFileName is null.</exception>
-        /// <exception cref="PathTooLongException">The specified archiveFileName exceeds the system-defined maximum length.
-        ///                                        For example, on Windows-based platforms, paths must be less than 248 characters,
-        ///                                        and file names must be less than 260 characters.</exception>
-        /// <exception cref="DirectoryNotFoundException">The specified archiveFileName is invalid, (for example, it is on an unmapped drive).</exception>
-        /// <exception cref="IOException">An unspecified I/O error occurred while opening the file.</exception>
-        /// <exception cref="UnauthorizedAccessException">archiveFileName specified a directory.
-        ///                                               -OR- The caller does not have the required permission.</exception>
-        /// <exception cref="FileNotFoundException">The file specified in archiveFileName was not found.</exception>
-        /// <exception cref="NotSupportedException">archiveFileName is in an invalid format. </exception>
-        /// <exception cref="InvalidDataException">The specified file could not be interpreted as a Zip file.</exception>
-        ///
-        /// <param name="archiveFileName">A string specifying the path on the filesystem to open the archive on. The path is permitted
-        /// to specify relative or absolute path information. Relative path information is interpreted as relative to the current working directory.</param>
+        /// <summary>Opens a zip archive for reading at the specified path.</summary>
+        /// <param name="archiveFileName">The path to the archive to open, specified as a relative or absolute path. A relative path is interpreted as relative to the current working directory.</param>
+        /// <returns>The opened zip archive.</returns>
+        /// <remarks><format type="text/markdown"><![CDATA[
+        /// This method is equivalent to calling the <xref:System.IO.Compression.ZipFile.Open%2A> method and setting the `mode` parameter to <xref:System.IO.Compression.ZipArchiveMode.Read>. The archive is opened with <xref:System.IO.FileMode.Open?displayProperty=nameWithType> as the file mode value. If the archive does not exist, a <xref:System.IO.FileNotFoundException> exception is thrown.
```

Comment:
Should the `///` line prefixes be included in CDATA sections?

Reply:
Yes.